.github/workflows/checks.yaml

```
@@ -0,0 +1,41 @@
+name: checks
+
+on:
+  workflow_call:
+    inputs:
+      publish:
+        required: false
+        type: string
+        default: "false"
+
+jobs:
+  check-style:
+    runs-on: ubuntu-latest
+    container:
+      image: rapidsai/ci-conda:latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Check style
+        run: "ci/check_style.sh"
+  conda-build:
+    needs: check-style
+    runs-on: ubuntu-latest
+    container:
+      image: rapidsai/ci-conda:latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Python build
+        run: "./ci/build_python.sh ${{ inputs.publish }}"
+  wheel-build:
+    needs: check-style
+    runs-on: ubuntu-latest
+    container:
+      image: "rapidsai/ci-wheel:latest"
```

Either add quotes everywhere or remove them everywhere.
```suggestion
      image: rapidsai/ci-wheel:latest
```